app/jobs/dossier_rebase_job.rb

```diff
@@ -0,0 +1,5 @@
+class DossierRebaseJob < ApplicationJob
```

Comment:
Si on enqueue tous ces jobs d'un coup, y'a sans doute des dossiers qui seront supprimés entre temps.

Je propose un :

```suggestion
class DossierRebaseJob < ApplicationJob
  # If by the time the job runs the Dossier has been deleted, ignore the rebase
  discard_on ActiveRecord::RecordNotFound
  
```